tests/src/test/scala/common/WskTestHelpers.scala

```diff
@@ -244,8 +243,8 @@ trait WskTestHelpers extends Matchers {
     entity: String,
     N: Int = 1,
     since: Option[Instant] = None,
-    pollPeriod: Duration = 1 second,
-    totalWait: Duration = 30 seconds)(check: Seq[ActivationResult] => Unit)(implicit wskprops: WskProps): Unit = {
+    pollPeriod: Duration = 1.second,
+    totalWait: Duration = 60.seconds)(check: Seq[ActivationResult] => Unit)(implicit wskprops: WskProps): Unit = {
```

Comment:
i think these changes are the result of a rebase.
they have no baring on this pr.